packages/typescript/WORKSPACE

```diff
@@ -23,6 +15,17 @@ yarn_install(
     yarn_lock = "//:yarn.lock",
 )
 
+# Uncomment for local development
+# local_repository(
+#     name = "build_bazel_rules_typescript",
+#     path = "../../../rules_typescript",
+# )
+http_archive(
```

Comment:
I wonder if we should just keep git since you only have to update the commit in one place rather than two

Reply:
git_repository flaked out on me in CI when it worked locally so I switched at that point. don't know why but it errored out with not being able to find a commit hash